etcdutl/snapshot/v3_snapshot.go

```diff
@@ -144,25 +144,25 @@ func (s *v3Manager) Status(dbPath string) (ds Status, err error) {
 			if b == nil {
 				return fmt.Errorf("cannot get hash of bucket %s", string(next))
 			}
-			if _, err := h.Write(next); err != nil {
+			if _, err = h.Write(next); err != nil {
 				return fmt.Errorf("cannot write bucket %s : %v", string(next), err)
 			}
 			iskeyb := (string(next) == "key")
-			if err := b.ForEach(func(k, v []byte) error {
-				if _, err := h.Write(k); err != nil {
-					return fmt.Errorf("cannot write to bucket %s", err.Error())
+			if berr := b.ForEach(func(k, v []byte) error {
```

Comment:
I think we can reuse the `err` here. We only need to rename it inside the closure function (line 152 and 155).
```suggestion
			if err = b.ForEach(func(k, v []byte) error {
```